src/Lykke.Service.Qtum.Api.Jobs/Modules/ServiceModule.cs

```diff
@@ -44,7 +47,15 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<AddressBalanceRepository>()
                 .As<IAddressBalanceRepository<AddressBalance>>()
                 .WithParameter(TypedParameter.From(_appSettings.Nested(s => s.QtumApiService.Db.DataConnString)));
-            
+
+            builder.RegisterType<AddressHistoryEntryRepository>()
+                .As<IAddressHistoryEntryRepository<AddressHistoryEntry>>()
+                .WithParameter(TypedParameter.From(_appSettings.Nested(s => s.QtumApiJobsService.Db.DataConnString)));
```

Comment:
I guess we may read DataConnString only once

Reply:
fixed